wordpress-mobile/WordPress-iOS
# Force specific .strings file to be interpreted as binary plists for Xcode 10 compatibility.

Looks like some of our `.strings` file are actually binary plists. This was fine in Xcode 9, but caused build failures under Xcode 10.

This diff overrides Xcode's default file-type detection and explicitly sets the file type as binary plists.

To test:
1. Build the app under Xcode 9
2. Verify that it still builds and no strings are missing (expecially in our extensions)
3. Build the app under Xcode 10 
2. Verify the strings as above
